gluten-ut/spark34/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/GlutenParquetRowIndexSuite.scala

```diff
@@ -315,10 +311,10 @@ class GlutenParquetRowIndexSuite extends ParquetRowIndexSuite with GlutenSQLTest
               // When there is no filter, the rowIdx values should be in range
               // [0-`numRecordsPerFile`].
               val expectedRowIdxValues = List.range(0, numRecordsPerFile)
-              assert(
-                dfToAssert
-                  .filter(col(rowIndexColName).isin(expectedRowIdxValues: _*))
-                  .count() == conf.numRows)
+              val df = dfToAssert
+                .select($"id")
```

Comment:
velox `row_index` generation logic would generate wrong indexs if without data colmn be selected (scan out).

checking this issue https://github.com/facebookincubator/velox/issues/9943

Reply:
@yma11 FYI

Reply:
Thanks for noting this out. Let's update this part once the fix merged.

Reply:
done